pkgs/by-name/sw/swagger-typescript-api/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  fetchFromGitHub,
+  yarn-berry_4,
+  stdenv,
+  nodejs,
+  makeWrapper,
+}:
+let
+  pname = "swagger-typescript-api";
+  version = "13.2.2";
+  yarn-berry = yarn-berry_4;
+in
+stdenv.mkDerivation (finalAttrs: {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "acacode";
+    repo = "swagger-typescript-api";
+    rev = version;
+    hash = "sha256-dpziQNXLwo2W+zZdFujaNufVoGg8M11G79LVWTi9G/M=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    yarn-berry.yarnBerryConfigHook
+    yarn-berry
+    nodejs
+  ];
+
+  missingHashes = ./missing-hashes.json;
+  offlineCache = yarn-berry.fetchYarnBerryDeps {
+    inherit (finalAttrs) src missingHashes;
+    hash = "sha256-8zL5rWzEQHY1XOrykoABRuYmfKPPc1YH3n0b10Bk1uI=";
```

```suggestion
    hash = "sha256-pD/yx1DyccEWczvDQM+gseravu8eIqumOBuG65htYr8=";
```
Think this changed since you fixed the final newline in missing-hashes.json.